### ISSUE TITLE
Crash when using /profile command from Just Enough Resources (issue cross-posted)

### ISSUE THREAD

**Issue by Crowfooted** [lumien231/Random-Things#343]
https://pastebin.com/v5uCNWW9

The command profiles chunks in the world (singleplayer only) to create a file detailing what ores are found in the world and where.

**Comment by lumien231** [comment]
Weirdly enough i can't reproduce this, does this also happen with only RT and JER installed?

**Comment by Crowfooted** [comment]
Haven't tested that yet, I'll give that a go.

Though now I think of it, it would be a little redundant if it worked with only those two installed since the point of the profile command is to profile modded ores in the world and where they appear, so it'd be pretty important for it to work with other mods in the pack.

**Comment by lumien231** [comment]
Yes, but if i can't reproduce it i can't figure out what causes it.

**Comment by Pival81** [comment]
any update on this?
